src/doc/contrib/src/team.md

```diff
@@ -58,12 +58,28 @@ Members are given privileges, such as:
 
 ### Meetings
 
-The team meets on a weekly basis on a video chat.
-If you are interested in participating, feel free to contact us on [Zulip].
+The primary focus of team meetings is for unblocking designs, including discussing
+- [RFCs][t-cargo-rfcs]
+- [FCPs][cargo-status-tracker]
+- [Items marked **I-nominated-to-discuss**][i-nominated]
+- [Items marked **S-needs-team-input**][s-team-input]
+- Topics proposed on [zulip][zulip-meeting], subject to availability
 
+Generally, topics for the agenda should be posted a day in advance to give people the opportunity to review them before the meeting.
+
+The team meets on a weekly basis ([ics][meeting-ics]) on a video chat.
+Members of the community are welcome to attend.
 Minutes for all meetings are recorded on [HackMD][meeting-minutes].
 
+For discussing contributions, [Office Hours](../process#mentorship) would be more appropriate venue.
```

Comment:
This helps if people read the markdown source from their editors or browsers, and want the link working.

```suggestion
For discussing contributions, [Office Hours](../process.md#mentorship) would be more appropriate venue.
```

Reply:
Has to be `process/index.md`.  It looks like mdbook just changes `.md` to `.html` and doesn't even check if the file was built.